kong/clustering/services/sync/rpc.lua

```diff
@@ -243,71 +294,24 @@ local function do_sync()
   -- { type = ..., entity = { ... }, version = "1", ws_id = ..., }
   for _, delta in ipairs(deltas) do
     local delta_version = delta.version
-    local delta_type = delta.type
     local delta_entity = delta.entity
-    local ev
+    local delta_type = delta.type
```

Comment:
This change seems to be unnecessary.

Reply:
My meaning is that it is same as line 246, why you changed the positioin?

Reply:
This if for style. Moving it here makes it more clear

Reply:
I don't think so, this change brings us any benefits except style clean? If no just keep it as it is.